swagger.json

```diff
@@ -4738,17 +4892,44 @@
                 "start_location_lng"
             ],
             "properties": {
-                "vehicle_id": {
-                    "type": "integer",
-                    "format": "int64",
-                    "description":  "ID of the vehicle assigned to the dispatch route. Note that vehicle_id and driver_id are mutually exclusive. If neither is specified, then the route is unassigned.",
-                    "example": 444
+                "vehicle": {
+                    "type": "object",
+                    "description": "Vehicle object",
```

Comment:
Same as above, could we define this in an `#/definitions/ExternalVehicle` object?